cmd/bosun/expr/funcs.go

```diff
@@ -344,6 +350,39 @@ func SeriesFunc(e *State, T miniprofiler.Timer, tags string, pairs ...float64) (
 	}, nil
 }
 
+func Crop(e *State, T miniprofiler.Timer, sSet *Results, startSet *Results, endSet *Results) (*Results, error) {
```

Comment:
Maybe I'm confused about the types here. The example makes it look like start and end are scalars, but here they are NumberSets.